package.json

```diff
@@ -30,7 +30,7 @@
     "es6-promise": "^4.1.1",
     "express": "4.16.x",
     "file-saver": "^1.3.3",
-    "fs-extra": "0.26.x",
+    "fs-extra": "^7.0.0",
```

Comment:
We were quite a few versions behind, which did not have the Promise based version for most of the functions, much less the utility `{read, write}Json` functions